README.md

```diff
@@ -36,7 +36,18 @@ ICICLE is a CUDA implementation of general functions widely used in ZKP. ICICLE
 
 ```sh
 mkdir -p build
-nvcc -o build/<ENTER_DIR_NAME> icicle/lib.cu -arch=native
+nvcc -o build/<ENTER_DIR_NAME> ./icicle/appUtils/ntt/ntt.cu ./icicle/appUtils/msm/msm.cu ./icicle/appUtils/vector_manipulation/ve_mod_mult.cu ./icicle/primitives/projective.cu -arch=native
+```
+
+### Testing the CUDA code
+
+We are using [googletest] library for testing. To build and run the test suite for finite field and elliptic curve arithmetic, run from the `icicle` folder:
+
+```sh
+mkdir -p build
```

Comment:
just fyi had issues building the pr due to conflicted versions of `nvcc` on my machine, I had to modify the path with an explicit version. 
```
//CUDA compiler
CMAKE_CUDA_COMPILER:FILEPATH=/usr/local/cuda-12.0/bin/nvcc
```
Maybe common problem - perhaps worth adding a troubleshooting section, not in this PR